torch/quantization/quantize.py

```diff
@@ -241,19 +255,22 @@ def quantize(model, run_fn, run_args, mapping=DEFAULT_MODULE_MAPPING):
         run_fn: a function for evaluating the prepared model, can be a
             function that simply runs the prepared model or a training loop
         run_args: positional arguments for `run_fn`
+        inplace: carry out model transformations in-place, the original model is destroyed
+        mapping: correspondence between original module types and quantized counterparts
 
     Return:
         Quantized model.
     """
 
-    model = copy.deepcopy(model)
+    if not inplace:
+        model = copy.deepcopy(model)
     model.eval()
-    prepare(model)
+    model = prepare(model, inplace=True)
```

Comment:
why do we need to assign with `inplace=True`?

Reply:
just for consistency with potential graph mode